2024-10-21 09:07:37,400 - app - ERROR - Exception on /containers [GET] [in /home/ec2-user/.local/lib/python3.9/site-packages/flask/app.py:1457]
Traceback (most recent call last):
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/home/ec2-user/aws_edfcatalogoqr/app/routes.py", line 324, in list_containers
    return render_template('list_containers.html', title='Mis Contenedores', containers=containers, form=form)
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/templating.py", line 147, in render_template
    return _render(
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/templating.py", line 128, in _render
    rv = template.render(context)
  File "/home/ec2-user/.local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/ec2-user/.local/lib/python3.9/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/ec2-user/aws_edfcatalogoqr/app/templates/list_containers.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ec2-user/aws_edfcatalogoqr/app/templates/layout.html", line 141, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ec2-user/aws_edfcatalogoqr/app/templates/list_containers.html", line 28, in block 'content'
    <a href="{{ url_for('main.edit_container', container_id=container.id) }}" class="btn btn-warning">Editar</a>
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/helpers.py", line 338, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/home/ec2-user/.local/lib/python3.9/site-packages/flask/helpers.py", line 325, in url_for
    rv = url_adapter.build(
  File "/home/ec2-user/.local/lib/python3.9/site-packages/werkzeug/routing.py", line 2315, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'main.edit_container' with values ['container_id']. Did you mean 'main.print_container' instead?
2024-10-21 09:26:18,038 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/aws_edfcatalogoqr/app/__init__.py:70]
2024-10-21 09:26:18,126 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/aws_edfcatalogoqr/app/models.py:23]
2024-10-21 09:26:18,160 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/aws_edfcatalogoqr/app/models.py:29]
2024-10-21 19:15:45,458 - app.utils - INFO - Correo de restablecimiento de contraseña enviado a edfrutos@gmail.com [in /home/ec2-user/aws_edfcatalogoqr/app/utils.py:164]
2024-10-21 19:16:03,252 - app.models - INFO - Token válido para el usuario ID: 66a21c1c106d5c0ec264b152 [in /home/ec2-user/aws_edfcatalogoqr/app/models.py:71]
2024-10-21 19:16:05,132 - app.models - INFO - Token válido para el usuario ID: 66a21c1c106d5c0ec264b152 [in /home/ec2-user/aws_edfcatalogoqr/app/models.py:71]
2024-10-21 19:16:16,974 - app.models - INFO - Token válido para el usuario ID: 66a21c1c106d5c0ec264b152 [in /home/ec2-user/aws_edfcatalogoqr/app/models.py:71]
2024-10-22 11:00:22,747 - app - INFO - Aplicación Flask iniciada correctamente [in /home/ec2-user/aws_edfcatalogoqr/app/__init__.py:70]
2024-10-22 11:00:22,826 - app.models - INFO - Desconexión exitosa de la base de datos anterior. [in /home/ec2-user/aws_edfcatalogoqr/app/models.py:23]
2024-10-22 11:00:22,862 - app.models - INFO - Conectado a la base de datos. [in /home/ec2-user/aws_edfcatalogoqr/app/models.py:29]
2024-10-22 11:37:17,492 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-22 11:37:17,492 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: 33fbbdebb46080a8.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-22 11:37:29,183 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-22 11:37:29,184 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: 33fbbdebb46080a8.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-22 11:37:40,062 - app - INFO - Iniciando actualización de cuenta para el usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:109]
2024-10-22 11:37:40,062 - app - INFO - Información recibida - Username: edefrutos, Email: edfrutos@gmail.com, Address: Calle Mango, 3, Phone: 983246972 [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:110]
2024-10-22 11:37:40,062 - app - INFO - Actualizando foto de perfil para el usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:115]
2024-10-22 11:37:40,117 - app.utils - INFO - Foto de perfil guardada en: /home/ec2-user/aws_edfcatalogoqr/app/static/profile_pics/d29e6cf78aae434b.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/utils.py:78]
2024-10-22 11:37:40,117 - app - INFO - Foto de perfil actualizada: d29e6cf78aae434b.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:119]
2024-10-22 11:37:40,182 - app - INFO - Cuenta actualizada exitosamente para el usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:137]
2024-10-22 11:37:40,373 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-22 11:37:40,373 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: d29e6cf78aae434b.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-22 16:18:50,698 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-22 16:18:50,698 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: d29e6cf78aae434b.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-22 16:22:03,798 - app.routes - INFO - Creación de contenedor iniciada. [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:228]
2024-10-22 16:22:04,044 - app.utils - INFO - Imagen del contenedor guardada en: /home/ec2-user/aws_edfcatalogoqr/app/static/container_pics/68b7fad74946acad.png [in /home/ec2-user/aws_edfcatalogoqr/app/utils.py:108]
2024-10-22 16:22:04,092 - app.routes - INFO - Contenedor creado con ID: 6717d12c86bd2465cd5ec7d5 [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:269]
2024-10-22 16:35:08,757 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-22 16:35:08,757 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: 640d3c88bdc46d9b.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-22 16:35:28,345 - app - INFO - Iniciando actualización de cuenta para el usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:109]
2024-10-22 16:35:28,345 - app - INFO - Información recibida - Username: edefrutos, Email: edfrutos@gmail.com, Address: Calle Mango, 3, Phone: 983246972 [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:110]
2024-10-22 16:35:28,345 - app - INFO - Actualizando foto de perfil para el usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:115]
2024-10-22 16:35:28,369 - app.utils - INFO - Foto de perfil guardada en: /home/ec2-user/aws_edfcatalogoqr/app/static/profile_pics/17f86b14e8993c4c.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/utils.py:78]
2024-10-22 16:35:28,369 - app - INFO - Foto de perfil actualizada: 17f86b14e8993c4c.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:119]
2024-10-22 16:35:28,429 - app - INFO - Cuenta actualizada exitosamente para el usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:137]
2024-10-22 16:35:28,672 - app - INFO - Cargando datos del usuario: edefrutos [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-22 16:35:28,673 - app - INFO - Mostrando imagen de perfil para el usuario edefrutos: 17f86b14e8993c4c.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-23 13:56:40,099 - app - INFO - Cargando datos del usuario: YuzоеHeisa [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-23 13:56:40,099 - app - INFO - Mostrando imagen de perfil para el usuario YuzоеHeisa: default.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-23 13:56:40,429 - app - INFO - Iniciando actualización de cuenta para el usuario: YuzоеHeisa [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:109]
2024-10-23 13:56:40,429 - app - INFO - Información recibida - Username: YuzоеHeisa, Email: yuz3@yzovka24.ru, Address: https://yuzovka.org, Phone: 86185622223 [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:110]
2024-10-23 13:56:40,488 - app - INFO - Cuenta actualizada exitosamente para el usuario: YuzоеHeisa [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:137]
2024-10-23 13:56:40,827 - app - INFO - Cargando datos del usuario: YuzоеHeisa [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:152]
2024-10-23 13:56:40,827 - app - INFO - Mostrando imagen de perfil para el usuario YuzоеHeisa: default.jpg [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:162]
2024-10-25 18:38:43,701 - app.routes - INFO - Creación de contenedor iniciada. [in /home/ec2-user/aws_edfcatalogoqr/app/routes.py:228]
2024-10-25 18:38:43,737 - app.utils - INFO - Imagen del contenedor guardada en: /home/ec2-user/aws_edfcatalogoqr/app/static/container_pics/3f404a27353585ee.jpeg [in /home/ec2-user/aws_edfcatalogoqr/app/utils.py:108]
